contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
I think we should really look at wkhtmltopdf because while TCPDF is probably the best pdf tool, it is not the best html2pdf. Maybe dompdf is better and it is in codebase already.